packages/diagram_capture/lib/diagram_capture.dart

```diff
@@ -146,7 +146,7 @@ class _DiagramTicker extends Ticker {
 
 // Provides a binding different from the regular Flutter binding so that
 // diagrams can control their timeline and physical device size.
```

Comment:
If you're going to make it "public", then you should probably change this to be a dartdoc comment.